src/dvclive/fastai.py

```diff
@@ -76,4 +77,7 @@ def after_fit(self):
         if _inside_fine_tune() and not self.freeze_stage_ended:
             self.freeze_stage_ended = True
         else:
+            if hasattr(self, "save_model"):
+                if self.save_model.last_saved_path:
+                    self.live.log_artifact(str(self.save_model.last_saved_path))
```

Comment:
How does fastai organize the saved models? Will the user end up with a new .dvc file for each call?

Reply:
Checking into the code it looks like the only problem would be if the user sets `every_epoch=True` (defaults to `False`). We could skip `log_artifact` if `every_epoch=True`

Reply:
Looking at the docs in https://docs.fast.ai/callback.tracker.html#savemodelcallback, should we be logging `learn.path/learn.model_dir` instead of a specific file?

Reply:
> Looking at the docs in https://docs.fast.ai/callback.tracker.html#savemodelcallback, should we be logging learn.path/learn.model_dir instead of a specific file? 

In cases where there is a single model, the `model_dir` would only contain `last_saved_path`. 

In the other case (`every_epoch=True`) I think logging the entire dir makes the downstream usage worse.

Reply:
🤔 It makes #140 more complex since we would have to figure out which file to load, right? Either way, hopefully it shouldn't matter to the user if we can make the integration work well. I'm surprised how little there is out there for resuming in situations like this.

> We could skip log_artifact if every_epoch=True

Do you mean calling `log_artifact` only at the end of the experiment? Or not calling it at all?

Reply:
> Do you mean calling log_artifact only at the end of the experiment? Or not calling it at all?

Not calling it at all, document it, and let the user decide what to pass to `log_artifact`.

I was playing around adapting https://github.com/iterative/example-get-started-experiments/ and I think I would actually want to call [`learn.export`](https://docs.fast.ai/learner.html#learner.export) after training and use `log_artifact` on its output.

You can use [`load_learner`](https://docs.fast.ai/learner.html#load_learner) on downstream usage but the files saved by `SaveModelCallback` require you to manually instantiate the learner and call `learn.load`, which is inconvenient.

Reply:
> I thought this was the preferred way for loading models in pytorch?

My understanding is that it is the best way to save the model in case one of the potential usages is resuming the training. 

---

Anyhow, the point is that I believe calling `log_artifact` with `every_epoch=True` is not really useful. If someone chooses to use the option, they can still use `log_artifact` manually.

I think by default we should only call `log_artifact` in the scenarios where we are sure is a single model file.

Reply:
After discussing with @daavoo, I think we should move forward with the current implementation that's in this PR for simplicity.